ibis/expr/operations/relations.py

```diff
@@ -587,7 +587,9 @@ class FillNa(Relation):
     """Fill null values in the table."""
 
     table: Relation
-    replacements: UnionType[Value[dt.Numeric | dt.String], FrozenDict[str, Any]]
```

Comment:
Could we separate this into two operations intead? Something like:
- `FillNa`
- `FillNaMapping`

Reply:
I'd prefer not to here.

Reply:
The changes here aren't significant enough to warrant this refactor, which is more annoying than it may seem :)